src/Scripts.purs

```diff
@@ -82,6 +84,10 @@ validatorHashEnterpriseAddress network valHash =
       , paymentCred: scriptHashCredential (unwrap valHash)
       }
 
+nativeScriptHashEnterpriseAddress :: NetworkId -> NativeScriptHash -> Address
```

Comment:
(basically something similar to what is done above)
```suggestion
-- | Converts a Plutus-style `NativeScriptHash` to an `Address` as an
-- | `EnterpriseAddress`.
nativeScriptHashEnterpriseAddress :: NetworkId -> NativeScriptHash -> Address
```

Reply:
There is no plutus-style NativeScriptHash, because plutus does not deal with native scripts.